server/models/searches.js

```diff
@@ -2,86 +2,36 @@ var mongoose = require('mongoose');
 	
 	
 var searchSchema = new mongoose.Schema({	
+    ids: {
+		type: [String]
+	},
     names: {
-		type: [[name]]
+		type: {
```

Comment:
I'm a bit unfamiliar with Mongoose. Will this support an array of names? Like searching times with multiple clinicians?

Reply:
You are correct, this needs to be an array of objects (and not a single object)